lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -163,16 +160,16 @@ def base_url
       end
 
       def avs_result(action, succeeded, response)
-        if succeeded
-          action == :purchase ? AVSResult.new(code: response["card"]["avsCheck"]) : nil
+        if succeeded && action == :purchase || :authorize
+          AVSResult.new(code: response['card']['avsCheck']) if response['card'] && response['card']['avsCheck']
         else
           nil
         end
       end
 
       def cvv_result(action, succeeded, response)
-        if succeeded
-          action == :purchase ? CVVResult.new(response["card"]["cvvCheck"]) : nil
+        if succeeded && action == :purchase || :authorize
```

Comment:
Same issue as line [the earlier line](https://github.com/activemerchant/active_merchant/pull/2650/files#diff-390e4ab1e508727fef10daef8ff96d35R163)